packages/connect-express/tsconfig.build.json

```diff
@@ -1,4 +1,7 @@
 {
-  "files": ["src/index.ts"],
-  "extends": "../../tsconfig.base.json"
+  "include": ["src/**/*.ts", "src/**/*.spec.ts"],
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true
+  }
```

Comment:
Do we really need this here? It wasn't necessary before.

Reply:
Yeah because now the `express-readme.spec.ts` file imports express and that file is included in our TS build process. The express middleware only has a type import for express, which was satisfied by `@types/express`.